PixiEditor/Helpers/Converters/FileExtensionToColorConverter.cs

```diff
@@ -7,30 +10,26 @@ namespace PixiEditor.Helpers.Converters
     public class FileExtensionToColorConverter :
         SingleInstanceConverter<FileExtensionToColorConverter>
     {
-        private static readonly SolidColorBrush PixiBrush = ColorBrush(226, 1, 45);
-
-        private static readonly SolidColorBrush PngBrush = ColorBrush(56, 108, 254);
-
-        private static readonly SolidColorBrush JpgBrush = ColorBrush(36, 179, 66);
+        private static readonly Dictionary<string, SolidColorBrush> extensions2Brushes;
 
         private static readonly SolidColorBrush UnknownBrush = ColorBrush(100, 100, 100);
 
+        static FileExtensionToColorConverter()
+        {
+            extensions2Brushes = new Dictionary<string, SolidColorBrush>();
+            extensions2Brushes[Constants.NativeExtension] = ColorBrush(226, 1, 45);
+            extensions2Brushes[SupportedFilesHelper.Format2Extension(ImageFormat.Png)] = ColorBrush(56, 108, 254);
+            extensions2Brushes[SupportedFilesHelper.Format2Extension(ImageFormat.Jpeg)] = ColorBrush(36, 179, 66);
+            extensions2Brushes[SupportedFilesHelper.Format2Extension(ImageFormat.Bmp)] = ColorBrush(40, 170, 236);
+            extensions2Brushes[SupportedFilesHelper.Format2Extension(ImageFormat.Gif)] = ColorBrush(180, 0, 255);
+        }
```

Comment:
Yes, but as far as I can tell, this only works for `.jpeg` but not for `.jpg`

Reply:
Yep, I'll fix it in the evening.

Reply:
Just did it along with a test for FileExtensionToColorConverter so that in the future we got notified in case of issues.